summary.md

```diff
@@ -1,5 +1,7 @@
-* [Assumptions](assumptions.md)
+* [Movies API example](movies/0.index.md)
```

Comment:
```suggestion
* [Assumptions](assumptions.md)
* [1 Entrypoint](api-documentation/1.entry-point.md)
  * [1.1 Security considerations](1.1.security-considerations.md)
* [2 API Documentation](api-documentation/2.api-documentation.md)
  * [2.1 Data structures](api-documentation/2.1.api-documentation-data-structures.md)
  * [2.2 User document](api-documentation/2.2.api-documentation-user-document.md)
  * [2.3 Complete API Documentation](api-documentation/2.3.complete-api-documentation.md)
* [Examples](examples/index.md)
  * [Movies API](examples/movies/0.index.md)
```

Reply:
This seams to be the case meanwhile

Reply:
I'm not sure I understand. I've made this suggestion to answer your question:

>I am not sure what you mean by that. Which items should be where? Are there broken links?

Reply:
The suggestion moves Assumptions under Drafts,  wich is the case meanwhile